commit bd4f6333ab054afca989dd28b67c144313ef0852 (refs/changes/87/87/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-07-15T23:01:35+00:00 (3 years, 3 months ago)
    
    MANATEE-310 manatee should not use pfexec (no-op) for zfs commands

diff --git a/package.json b/package.json
index 239c086..96b435d 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "carrier": "0.2.0",
-        "manatee": "git+ssh://git@github.com:joyent/manatee.git#ee1b40a555d24c9337b29db865a46e9437442233"
+        "manatee": "git+ssh://git@github.com:joyent/manatee.git#3771c6fbd979ad41e7a71b19f9e2b7ff99542134"
     },
     "sdcDependencies": {
         "config-agent": ">=1.2.0"
